bokehjs/src/lib/models/glyphs/image_url.ts

```diff
@@ -74,6 +75,32 @@ export class ImageURLView extends XYGlyphView {
       }
       img.src = this._url[i]
     }
+
+    const xs : Array<number> = []
+    const ys : Array<number> = []
+    for (var i=0; i < this._x.length; i++) {
+      xs.push(this._x[i])
+      ys.push(this._y[i])
+    }
+
+    // if the width/height are in screen units, don't try to include them in bounds
+    if (this.model.properties.w.units == "data") {
+      for (var i=0; i < this._x.length; i++) {
```

Comment:
`var` isn't allowed across the code base. `let` should be used instead. This is enforced by tslint (see above).

Reply:
Also, whitespace.